<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Temperature resoluiton tester</string>
  </property>
  <widget class="PlotWidget" name="graphicsViewTemperaturePlot">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>531</width>
     <height>411</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>771</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Status update</string>
   </property>
   <widget class="QTextEdit" name="textEditStatusUpdate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>741</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>231</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Sensibo API</string>
   </property>
   <widget class="QPushButton" name="pushButtonInitialCheck">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Check avaiable Daikin devices.</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditAPIKey">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>bUBfc0Vz8bQF85dnwxIQw8PrVlZsA2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>API Key</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Device name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_DeviceName"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>231</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Test parameters</string>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton_StartMeasurement">
       <property name="text">
        <string>Start!</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_StopMeasurement">
       <property name="text">
        <string>Stop!</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>211</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>T1 (C)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Max Period for T1 (mins)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>T2 (C)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Period for T2 (mins)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Time step size (mins)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Measurement time (mins)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="lineEdit_T1"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_T1PeriodMins"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_T2"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_T2PeriodMins"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_TimeStepSizeMins"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_MeasurementTimePeriodMins"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
